src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -124,6 +124,12 @@ private int RunExecutable()
                 return result;
             }
 
+            // Add Nuget Packages Probing Path
+            var nugetPackagesRoot = _context.PackagesDirectory;
```

Comment:
Intention here was to give something to quickly test the scenario.

As this fleshes out in the host I'll make better investments here.